File: output/ChatGPT/codenet/Java/C++/s151517960.cpp
#include <iostream>
#include <unordered_map>
using namespace std;

int main() {
  int n;
  cin >> n;
  unordered_map<int, int> map;
  for(int i = 0; i < n; i++) {
    int a;
    cin >> a;
    if(map.count(a)) {
      map[a] += 1;
    } else {
      map[a] = 1;
    }
  }
  int ans = 0;
  for(auto it = map.begin(); it != map.end(); it++) {
    if((it->second % 2) == 1) ans++;
  }
  cout << ans << endl;

  return 0;
}

File: output/ChatGPT/codenet/Java/Python/s151517960.py
n = int(input())
map = {}
for i in range(n):
  a = int(input())
  if a in map:
    map[a] += 1
  else:
    map[a] = 1
ans = 0
for key in map:
  if map[key] % 2 == 1:
    ans += 1
print(ans)